graphql_compiler/tests/test_compiler.py

```diff
@@ -4120,22 +4143,23 @@ def test_fold_on_output_variable(self):
                 )
             ])}
         '''
-        expected_sql = '''
+        expected_mssql = SKIP_TEST
+        expected_postgresql = '''
             SELECT
-                [Animal_1].name AS animal_name,
+                "Animal_1".name AS animal_name,
                 coalesce(folded_subquery_1.fold_output_name, ARRAY[]::VARCHAR[]) AS child_names_list
             FROM
-                db_1.schema_1.[Animal] AS [Animal_1]
+                "db_1.schema_1"."Animal" AS "Animal_1"
```

Comment:
In MSSQL `db_1` is the database and `schema_1` is the schema. Postgres doesn't have the concept of multiple databases on the same server, so the whole thing `db_1.schema_1` is just the name of the schema. This will probably confuse anyone reading this. It will get even more confusing when creating postgres integration tests, and we initialize schems with names `db_1.schema_1`.

It might be worth just not specifying `db_1` in the `postgresql_schema_info`. Simply have `schema_1` as the schema name. MSSQL and Postgres are two different databases, and our schema setups for both of them don't have to be identical. In fact they might further diverge as we start testing dialect-specific features, like array types, or inheritance.